app/code/Magento/Captcha/Controller/Refresh/Index.php

```diff
@@ -1,42 +1,46 @@
 <?php
```

Comment:
We are going to have conflicts in this file, as I changed this also for my PR #27381